src/app.ts

```diff
@@ -1,6 +1,72 @@
+type ResultData = {
+    id: string,
+    gender: string,
+    age: number,
```

Comment:
Codacy has a fix for the issue: [Replace `····age:·number,` with `··age:·number;`](https://app.codacy.com/gh/dekunleroy/pipeline_assessment/pullRequest?prid=9962768)
```suggestion
  age: number;
```